pkl-codegen-java/src/main/kotlin/org/pkl/codegen/java/Main.kt

```diff
@@ -100,13 +100,13 @@ class PklJavaCodegenCommand :
       The specified annotation type must be annotated with `@java.lang.annotation.Target(ElementType.TYPE_USE)`
       or the generated code may not compile.
     """
-          .trimIndent()
+          .trimIndent(),
     )
 
   private val implementSerializable: Boolean by
     option(
         names = arrayOf("--implement-serializable"),
-        help = "Whether to generate classes that implement java.io.Serializable."
+        help = "Whether to generate classes that implement java.io.Serializable.",
```

Comment:
Spotless Kotlin is a little funky; touching a single file will add trailing commas everywhere